src/OrchardCore.Modules/OrchardCore.Media/Services/SecureMediaMarkerService.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace OrchardCore.Media.Services
+{
+    internal sealed class SecureMediaMarkerService { }
```

Comment:
Why do we need this?

Reply:
It's a quick way to detect if the feature is enabled.

Reply:
If it's a marker I prefer to name it `SecureMediaMarker` instead coz nothing related to the services

Reply:
I named that after the existing https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore/OrchardCore.Abstractions/Shell/Distributed/DistributedShellMarkerService.cs

Reply:
I'm suggest to rename that too :)

Reply:
I renamed the class and also moved `SecureMediaExtensions` to a separate file.